### DISCUSSION TITLE
Layout used in demo

### DISCUSSION THREAD

**Discussion by kreativmind** [automadcms/discussions#44] [General]
I am wondering if you can tell me how you did the layout for the demo. I want to have the same exact look and functionality as my site is nothing more than uploading an image of a design. Do you have a template I can use?

**Comment by marcantondahmen** [comment]
Hi there, the demo just uses the default theme. You can see how it works by editing the main section of the landing page. It is basically a layout section block with a slider and a gallery block inside.
